### PR TITLE
Allure report

### PR DESCRIPTION
**Pull request by avoloshchuk567**
Allure report was added

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,13 @@
 
     <properties>
         <seleniumJavaVersion>3.141.59</seleniumJavaVersion>
+        <seleniumChromeDriverVersion>3.141.59</seleniumChromeDriverVersion>
         <testngVersion>6.14.3</testngVersion>
+        <maven.compiler.target>1.10</maven.compiler.target>
+        <maven.compiler.source>1.10</maven.compiler.source>
+        <allureTestngVersion>2.13.1</allureTestngVersion>
+        <mavenSurefirePluginVersion>3.0.0-M4</mavenSurefirePluginVersion>
+        <aspectj.version>1.9.5</aspectj.version>
```
Comment (uvidit) [inline pom.xml]: I suggest being more consistent in naming.
I mean if you already used **+'Version'** then keep do it and don't switch to a bit different **+'.version'**

File: src/test/java/signUpTesting/VerifySignUpLink.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    @Step("Check that 'Sign up' link is present")
+    public void verifySignUpLinkPresent() {
+        driver.findElement(By.xpath("//a[text()='Sign up']")).isDisplayed();
+    }
+
+    @Test(description = "Verification of 'Sign up' link title correctness", dependsOnMethods = {"verifySignUpLinkPresent"})
+    @Description("Test description: check that 'Sign up' title is correct")
+    @Step("Check that 'Sign up' link title is correct")
+    public void verifySignUpLinkCorrectTitle() {
+        String signUpLink = driver.findElement(By.xpath("//a[text()='Sign up']")).getText();
```
Comment (uvidit) [inline src/test/java/signUpTesting/VerifySignUpLink.java]: The variable represent signUpLinkText , but not signUpLink... please be more precise while naming. 
Thats why your next assert a bit confused me at first.

**Review by uvidit — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (avoloshchuk567) [reply pom.xml]: Changed to aspectjVersion in commit
30dc54fd9e13c250acf9eb8e72598d0ee2d4d232

File: src/test/java/signUpTesting/VerifySignUpLink.java (hunk 2 above)
Comment (avoloshchuk567) [reply src/test/java/signUpTesting/VerifySignUpLink.java]: Variable name is changed to signUpLinkText in commit
30dc54fd9e13c250acf9eb8e72598d0ee2d4d232

**Pull request merged**